src/components/screens/Settings/Appearance/AppearanceScreen.tsx

```diff
@@ -187,7 +187,10 @@ function AppearanceScreen({ navigation }: IProps) {
           </CSection>
         )}
 
-        <CSection header={t("settings.appearance.gestures.header")}>
+        <CSection
+          header={t("settings.appearance.gestures.header")}
+          footer="Disabling swipe to vote will allow for full-screen swipe gestures."
```

Comment:
Can this text be moved/added to the translation file?

Reply:
Oh yea...Keep forgetting about this, great catch!